osu.Game/Screens/Select/FilterCriteria.cs

```diff
@@ -46,6 +46,7 @@ public class FilterCriteria
         };
 
         public OptionalTextFilter[] SearchTerms = Array.Empty<OptionalTextFilter>();
+        public OptionalTextFilter[] DifficultySearchTerms = Array.Empty<OptionalTextFilter>();
```

Comment:
If this is to even exist, it needs xmldoc. Things are getting too confusing to follow.